examples/electron/package.json

```diff
@@ -45,30 +41,88 @@
     "@theia/preferences": "^0.14.0",
     "@theia/preview": "^0.14.0",
     "@theia/process": "^0.14.0",
-    "@theia/python": "^0.14.0",
     "@theia/scm": "^0.14.0",
     "@theia/search-in-workspace": "^0.14.0",
     "@theia/task": "^0.14.0",
     "@theia/terminal": "^0.14.0",
-    "@theia/textmate-grammars": "^0.14.0",
-    "@theia/tslint": "^0.14.0",
     "@theia/typehierarchy": "^0.14.0",
-    "@theia/typescript": "^0.14.0",
     "@theia/userstorage": "^0.14.0",
     "@theia/variable-resolver": "^0.14.0",
     "@theia/workspace": "^0.14.0"
   },
   "scripts": {
     "prepare": "yarn run clean && yarn build",
     "clean": "theia clean",
-    "build": "theiaext compile && theia build --mode development",
+    "build": "theiaext compile && theia build --mode development && yarn postinstall",
     "watch": "concurrently -n compile,bundle \"theiaext watch --preserveWatchOutput\" \"theia build --watch --mode development\"",
     "start": "theia start --plugins=local-dir:../../plugins",
     "start:debug": "yarn start --log-level=debug",
     "test": "electron-mocha --timeout 60000 \"./lib/test/**/*.espec.js\"",
-    "test:ui": "wdio wdio.conf.js"
+    "test:ui": "wdio wdio.conf.js",
+    "postinstall": "download-debug-adapters"
   },
   "devDependencies": {
     "@theia/cli": "^0.14.0"
+  },
+  "adapterDir": "../../plugins",
```

Comment:
It would be nice to find a way to avoid duplication. Clients who distribute for browser and electron will have the same issue.